docs/source/tutorials/image.ipynb

```diff
@@ -39,9 +34,9 @@
     "from cleanlab import Datalab\n",
     "\n",
     "lab = Datalab(data=your_dataset, label_name=\"column_name_of_labels\")\n",
-    "lab.find_issues(pred_probs=your_pred_probs, issue_types={\"label\":{}})\n",
+    "lab.find_issues(pred_probs=pred_probs, features=features)\n",
```

Comment:
aren't you missing the image_key here?

Reply:
I didn't add it purposefully, because it works only for huggingface dataset and not for other formats.